processor/src/chiplets/mod.rs

```diff
@@ -115,11 +115,11 @@ mod tests;
 #[derive(Debug)]
 pub struct Chiplets {
     /// Current clock cycle of the VM.
-    clk: RowIndex,
-    hasher: Hasher,
-    bitwise: Bitwise,
-    memory: Memory,
-    kernel_rom: KernelRom,
+    pub clk: RowIndex,
```

Comment:
We can get rid of the `clk` field, and replace everywhere that it is used with `process.system.clk()` instead